app/presenters/tslr_claim_csv_row.rb

```diff
@@ -0,0 +1,45 @@
+require "delegate"
+
+class TslrClaimCsvRow < SimpleDelegator
+  def csv_row
```

Comment:
rename this method to `data`? calling `tslr_claim_csv_row.data` feels more natural